proto/poktroll/application/query.proto

```diff
@@ -53,6 +53,8 @@ message QueryGetApplicationResponse {
 
 message QueryAllApplicationsRequest {
   cosmos.base.query.v1beta1.PageRequest pagination = 1;
+  // delegatee_gateway_address, if specified, filters the application list to only include those with delegation to the specified gateway address.
+  string delegatee_gateway_address = 2;
```

Comment:
Thoughts on renaming this to `gateway_address_delegated_to`.

I realize that `delegatee_gateway_address` more dramatically correct (probably?) but it confuses me when I read it and forces me to think.

When I read `gateway_address_delegated_to`, I don't need to think.

Reply:
100% agreed.

TODO_MAINNET instead of beta though ;)

Reply:
Done, thank you for the reviews.